chat-plugins/blackjack.js

```diff
@@ -0,0 +1,402 @@
+/**
+ * Blackjack Game
+ * Pokemon Showdown - http://pokemonshowdown.com/
+ *
+ * This allows users to play blackjack.
+ * Credits: jd, panpawn
+ * Card images by Byron Knoll
+ *
+ * @license MIT license
+ */
+'use strict';
+
+const IMAGES = 'http://ps-dev-panpawn1.c9users.io/cards/'; // image dir here pliz
+
+class Blackjack extends Rooms.RoomGame {
+	constructor(room, user, target) {
+		super(room);
+		this.room = room;
+		this.turnTimeoutMinutes = 1;
+		this.startedBy = user.name;
+		this.allowRenames = true;
+		this.playerCap = 2; // to change to... 6?
+		this.minimumPlayers = 2;
+		this.spectators = Object.create(null);
+
+		this.dealer = {
+			cards: [],
+			points: 0,
+			name: 'The Dealer',
+		};
+
+		this.symbols = {
+			'♥': 'H',
+			'♦': 'D',
+			'♣': 'S',
+			'♠': 'C',
+		};
+		this.deck = Dex.shuffle([
+			'A♥', 'A♦', 'A♣', 'A♠', '2♥', '2♦', '2♣', '2♠', '3♥', '3♦', '3♣',
+			'3♠', '4♥', '4♦', '4♣', '4♠', '5♥', '5♦', '5♣', '5♠', '6♥', '6♦', '6♣', '6♠',
+			'7♥', '7♦', '7♣', '7♠', '8♥', '8♦', '8♣', '8♠', '9♥', '9♦', '9♣', '9♠', '10♥',
+			'10♦', '10♣', '10♠', 'J♥', 'J♦', 'J♣', 'J♠', 'Q♥', 'Q♦', 'Q♣', 'Q♠', 'K♥', 'K♦',
+			'K♣', 'K♠',
+		].splice(0));
+
+		this.id = this.room.id;
+		this.gameid = Math.floor(Math.random() * 100000000);
```

Comment:
Why not use the number of games in the room as all of the other game plugins do?